v3.1/how-to/maintain/backup-and-restore/br.md

```diff
@@ -8,6 +8,95 @@ category: how-to
 
 Backup & Restore (BR) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with [`mydumper`/`loader`](/v3.1/how-to/maintain/backup-and-restore/mydumper-loader.md), BR is more suitable for scenarios of huge data volume. This document describes the BR command line, detailed use examples, best practices, restrictions, and introduces the implementation principles of BR.
 
+## Usage restrictions
+
+- BR only supports TiDB v3.1 and later versions.
+- Currently, TiDB does not support backing up and restoring partitioned tables.
+- Currently, you can perform restoration only on new clusters.
+- Multiple backup operations are recommended to be executed serially, otherwise different backup operations might interfere with each other.
```

Comment:
```suggestion
- It is recommended that you execute multiple backup operations serially. Otherwise, different backup operations might interfere with each other.
```